modules/social_features/social_group/modules/social_group_quickjoin/src/Form/SocialGroupQuickjoinSettings.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace Drupal\social_group_quickjoin\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\group\Entity\GroupType;
+
+/**
+ * Class SocialEventTypeSettings.
+ *
+ * @package Drupal\social_group_quickjoin\Form
+ */
+class SocialGroupQuickjoinSettings extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return [
+      'social_group_quickjoin.settings',
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'social_group_quickjoin_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('social_group_quickjoin.settings');
+
+    $form['help'] = [
+      '#type' => 'item',
+      '#markup' => $this->t("Enabling this feature gives site builders the possibility to create group quickjoin links (EG: /group/1/quickjoin). Furthermore it's possible to skip the confirmation step on a group type basis."),
+    ];
+
+    $form['social_group_quickjoin_enabled'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t("Enable quickjoin"),
+      '#description' => $this->t("Allow users to join groups with a singe click."),
```

Comment:
@jochemvn there's a typo in this string.

```suggestion
      '#description' => $this->t("Allow users to join groups with a single click."),
```

Reply:
Applied!